addons/binding/org.openhab.binding.network/ESH-INF/thing/thing-types.xml

```diff
@@ -45,9 +45,17 @@
 				<label>Use System Ping</label>
 				<description>Uses the ping program from the system</description>
 				<default>false</default>
-			</parameter>		
+			</parameter>
+			<parameter name="arping" type="boolean">
+				<label>Use arping</label>
+				<description>Use arping instead of regular ping. Only works on Unix and arping needs to be installed. When using arping it is required to specify a network interface.</description><default>false</default>
```

Comment:
You should probably say "Unix-like" systems since it includes Linux, Mac OSX and so on. You should also mention that arping requires root privileges (at least on Linux).